src/gup.rc

```diff
@@ -50,35 +50,34 @@ BEGIN
 END
 
 IDD_PROGRESS_DLG DIALOGEX 0, 0, 285, 39
-STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_BORDER | WS_CAPTION //| WS_SYSMENU
-EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
+STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_BORDER | WS_CAPTION
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
-	PUSHBUTTON      "Cancel",IDCANCEL,227,11,50,14
+    PUSHBUTTON      "Cancel",IDCANCEL,227,11,50,14
 END
 
 IDD_PROXY_DLG DIALOGEX 0, 0, 224, 84
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION
-EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
 CAPTION "Proxy Settings"
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     RTEXT           "Proxy server : ",IDC_PROXYSERVER_STATIC,11,18,61,8
     EDITTEXT        IDC_PROXYSERVER_EDIT,77,15,86,14,ES_AUTOHSCROLL
-	RTEXT           "Port : ",IDC_PORT_STATIC,11,41,61,8
-    EDITTEXT        IDC_PORT_EDIT,77,39,34,14,ES_NUMBER 
-	PUSHBUTTON      "OK",IDOK,55,66,50,14
+    RTEXT           "Port : ",IDC_PORT_STATIC,11,41,61,8
+    EDITTEXT        IDC_PORT_EDIT,77,39,34,14,ES_NUMBER
+    PUSHBUTTON      "OK",IDOK,55,66,50,14
     PUSHBUTTON      "Cancel",IDCANCEL,134,66,50,14
 END
 
-IDD_YESNONEVERDLG DIALOGEX 0, 0, 285, 94
-STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
+IDD_YESNONEVERDLG DIALOGEX 0, 0, 230, 86
+STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION
 CAPTION "Update available"
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
-    LTEXT           "An update package is available, do you want to download it?",IDC_YESNONEVERMSG,13,20,300,16
-    PUSHBUTTON   "Yes",IDYES,50,60,50,14
-    PUSHBUTTON   "No",IDNO,110,60,50,14
-    PUSHBUTTON   "Never",IDCANCEL,170,60,50,14
+    LTEXT           "An update package is available, do you want to download it?",IDC_YESNONEVERMSG,10,10,210,16

```

Comment:
Please keep the width as large as possible, because of the translated text variable length.

Reply:
I feel this is not required. Because this is multi line control...
Now, I made `IDC_CHK_IN_BACKGROUND` also multiple line, so that long translation will be handled automatically. 

![image](https://user-images.githubusercontent.com/14791461/47165899-a1add500-d318-11e8-9c8d-dcac0b272923.png)